pb/transaction.go

```diff
@@ -114,6 +114,14 @@ func (m *Subscribe) ToMap() map[string]interface{} {
 	}
 }
 
+func (m *Unsubscribe) ToMap() map[string]interface{} {
+	return map[string]interface{}{
+		"subscriber": common.BytesToUint160(m.Subscriber),
```

Comment:
common.BytesToUint160 has been changed to common.HexStr for subscribe, we should also change it here (after rebase)

Reply:
done